services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/core/docker_utils.py

```diff
@@ -236,3 +239,23 @@ async def _pull_image_with_progress(
             await progress_cb(total_current, total_total)
 
         await log_cb(f"pulling {shorter_image_name}: {pull_progress}...")
+
+
+@cached()
+async def supports_volumes_with_quota() -> bool:
+    async with docker_client() as docker:
+        docker_volume: Optional[DockerVolume] = None
+        try:
+            docker_volume = await docker.volumes.create(
+                {
+                    "name": f"check-quota-{uuid4()}",
+                    "Driver": "local",
+                    "DriverOpts": {"size": "1m"},
+                }
+            )
+        except aiodocker.DockerError:
+            logger.debug("No support for volume with quota")
```

Comment:
you should distinguish between an error because quotas really cannot be created and a completely different error thrown by docker client/daemon